commonwealth-vlr-engine.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'iiif-presentation', '~> 0.2.0'
   s.add_dependency 'madison', '~> 0.5.0'
   s.add_dependency 'rails', '~> 6.0.5'
```

Comment:
Change to ` s.add_dependency 'rails', '~> 6.0.6'`

Reply:
Also could you add `gem "json", ">= 2.3.0"` to the `Gemfile` so we can finally quash this `dependabot` alert please